### ISSUE TITLE
[Bug] AUTO_DB_BACKUP_KEEP not working properly

### ISSUE THREAD

**Issue by johnny-tknoska** [leiweibau/Pi.Alert#310]
### Is there an existing issue for this?

- [X] I have searched the existing issues and the Help/FAQ section of Pi.Alert

### Current Behavior

First of all, happy 2-year anniversary of this great project! Congratulations! 

There's a bug with the AUTO_DB_BACKUP_KEEP setting. It doesn't work well for DB file backups; it works well for .conf file backups. 
I have this setting in pialert.conf:
AUTO_DB_BACKUP = True
AUTO_DB_BACKUP_CRON = '3 19 */2 * *'
AUTO_DB_BACKUP_KEEP = 9

The backups work fine for both DB backup & .conf backup as you can see from the attached screenshots, but the AUTO_DB_BACKUP_KEEP is not working properly as it only keeps 5 copies of DB files instead of 9....for pialert.conf it works fine keeping 9 copies as expected. 

### Environment

OS: Raspberry Pi OS 12 bookworm , Native
Linux 6.6.28+rpt-rpi-v8 #1 SMP PREEMPT Debian 1:6.6.28-1+rpt1 (2024-04-22) aarch64

### Anything else?

**Pi.Alert .conf backups (9 files OK as expected):**

<img width="643" alt="pialert conf backups" src="https://github.com/leiweibau/Pi.Alert/assets/160315398/f783ed7d-092b-4042-bdd5-e9bf36830bed">

**Pi.Alert DB backups (just 5 files, not 9 as expected):**

<img width="675" alt="pialert db backups" src="https://github.com/leiweibau/Pi.Alert/assets/160315398/8668c90f-9981-4a04-8dcc-3529bae456e8">

**Comment by leiweibau** [comment]
Yes, the 5 is still hard-coded in the DB backups. Will be fixed with the next update. Thanks for the report

**Comment by leiweibau** [comment]
Update released with https://github.com/leiweibau/Pi.Alert/commit/530948a7e9b2b0d78d3168e44ec55f68d0417b0c
